### PR TITLE
V7 - minor fixes

### PR DESCRIPTION
**Pull request by Aaron128l**
- Realign Badges & Moderator Icons
- Fixes Deleted Messages

### CODE REVIEW

File: src/modules/chat_deleted_messages/index.js [hunk 1]
```diff
@@ -38,11 +38,10 @@ function onClearChat(name, tags) {
                 });
 
                 if (!settings.get('showDeletedMessages')) {
-                    $message.addClass('bttv-click');
                     const $messageClone = $message.clone();
+                    $message.addClass('bttv-click');
                     $message.click(() => {
                         $message.replaceWith($messageClone);
-                        $message.removeClass('bttv-click');
```
Comment (night) [inline src/modules/chat_deleted_messages/index.js]: why is this removed?

File: src/css/betterttv.css [hunk 2]
```diff
@@ -2347,6 +2350,8 @@ body.channel_new.columns.ember-application {
     cursor: pointer;
 }
 
-.chat-line .bttv-deleted {
+.chat-line .bttv-deleted.bttv-click {
     opacity: 0.5;
+    text-decoration: none;
```
Comment (night) [inline src/css/betterttv.css]: i prefer the underline

File: src/css/betterttv.css [hunk 3]
```diff
@@ -157,8 +153,16 @@ input.text:focus,select:focus,textarea:focus {
    text-decoration: none;
 }
 
-.ember-chat .badges, .ember-chat .mod-icons {
-    font-size: 0;
+.ember-chat .badges span:last-child .badge {
```
Comment (night) [inline src/css/betterttv.css]: the last time you messed with badges you caused a lot of alignment issues when there were:

1) no mod icons
2) no badges
3) no timestamps
4) mix-match 1-3

File: src/css/betterttv.css (hunk 2 above)
Comment (night) [inline src/css/betterttv.css]: also this breaks opacity of deleted messages with show deleted messages on

File: src/modules/chat_deleted_messages/index.js (hunk 1 above)
Comment (Aaron128l) [inline src/modules/chat_deleted_messages/index.js]: I switch the order of the clone creation so the `bttv-click` class is added **after** the clone is created. No need to remove the class, since it wasn't there when the clone was created.

File: src/css/betterttv.css (hunk 2 above)
Comment (Aaron128l) [inline src/css/betterttv.css]: This only affects the opacity of messages with `bttv-click`. With 'Show Delete Messages' enabled the class isn't added, showing the normal message with no opacity.

Comment (night) [inline src/css/betterttv.css]: `.bttv-deleted` is added when a message is deleted.. you're removing its class

File: src/css/betterttv.css (hunk 3 above)
Comment (Aaron128l) [inline src/css/betterttv.css]: All alignment issues stemmed from `font-size: 0` on both the mod icons and badges. With it being removed, the chat line goes back to its former alignment. All vertical margin tweaking is because of the compensation for my mistake. This change is mainly for the left and right additional whitespace added with `span` in an `inline-block` display. 

While I didn't want to resort to margins. `font-size: 0` was a mistake on multiple fronts.

File: src/css/betterttv.css (hunk 2 above)
Comment (Aaron128l) [inline src/css/betterttv.css]: `.bttv-deleted` is always on a deleted message, yes. I'm only applying the opacity to the `<message deleted>` text applied with the `.bttv-click` class. 

I believe for features like Show Delete Messages a deleted message shouldn't be opaque. It's open for personal preference.
